Cooper/ClientApp/src.v2/app/components/login-form/login-form.component.html

```diff
@@ -1,25 +1,29 @@
-<div align="center">
+<div align="center" >
 
   <form class="coop-form personal-data-form" #loginForm="ngForm" (ngSubmit)="onSignInClicked(loginForm)">
-
-
-    <div class="div--color div--positioning div--font">Sign In</div>
+     
+    
+    <div class="div--color div--positioning div--font">{{'LOGIN-FORM.TITLE' | translate}}
```

Comment:
we don't use -- in class names

Reply:
it`s not mine, but i`ll fix it and remember :D